src/content_script/misskeyInject.js

```diff
@@ -0,0 +1,148 @@
+"use strict";
+
+/**
+ * Replacement onClick handler for interaction buttons.
+ * 
+ * @param {Event} event
+ * @returns {void}
+ */
+function onClickInteract(event) {
+    event.stopImmediatePropagation();
+    event.stopPropagation();
+    event.preventDefault();
+    const articleElement = event.target.closest("article");
+    let headerElement = (
+        articleElement === null
+            ? null
+            : articleElement.querySelector("header > div > a") // misskey
+    );
+    headerElement = (
+        (articleElement !== null && headerElement === null)
+            ? articleElement.querySelector(".created-at") // calckey
+            : headerElement
+    );
+    const tootId = (
+        headerElement === null
+            ? window.location.pathname.toString()
+            : headerElement.getAttribute("href")
+    );
+    // activate AutoRemoteFollow
+    window.open(`${tootId}#interact`, "_blank");
+}
+
+/**
+ * Wait for element to appear.
+ * 
+ * @param {string} selector
+ * @param {boolean} [multiple=false]
+ * @param {number} [timeoutDuration=200000]
+ * @see {@link https://github.com/storybookjs/test-runner/blob/6d41927154e8dd1e4c9e7493122e24e2739a7a0f/src/setup-page.ts#L134}
+ *  from which this was adapted
+ * @returns {Promise}
+ */
+function waitForElement(selector, multiple = false, timeoutDuration = 200000) {
```

Comment:
Hmm did not we have a very similar code in the Mastodon injection code? Maybe we can generalize/have that in some `common.js` or so for all injection scripts?